.gitignore

```diff
@@ -6,3 +6,5 @@
 venv/
 .pytest_cache/
 **__pycache__/
+result.csv
```

Comment:
The `result.csv` file is being ignored. Ensure this is intentional, as it might be a necessary part of your project. If it's a generated file or contains sensitive data, it's fine to ignore it.